docs/reference/docs/multi-termvectors.asciidoc

```diff
@@ -74,9 +125,12 @@ POST /twitter/_mtermvectors
 --------------------------------------------------
 // TEST[setup:twitter]
 
-Additionally, just like for the <<docs-termvectors,termvectors>>
-API, term vectors could be generated for user provided documents.
-The mapping used is determined by `_index`.
+[[docs-multi-termvectors-artificial-doc]]
+[float]
```

Comment:
I don't believe this `[float]` is needed.
```suggestion
```

Reply:
There are lots of unnecessary floats in the ref docs.